server/views/cms/rest.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from flask import jsonify, request
+import requests
+import html
+
+from server import app
+from server.util.request import api_error_handler
+
+logger = logging.getLogger(__name__)
+BASE_URL = app.config['PAYLOAD_API_URL']
+API_KEY = app.config['PAYLOAD_API_KEY']
```

Comment:
If we have logging or sentry then I think we should error if either `URL` or `KEY` is missing.